tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MatchTypeSuite.scala

```diff
@@ -222,7 +222,7 @@ class MatchTypeSuite extends BaseDottySuite {
                 Type.Name("Left"),
                 List(
                   TypeCase(Type.Name("Unit"), Type.Name("Right")),
-                  TypeCase(Type.Wildcard(Type.Bounds(None, None)), Type.Name("Left"))
+                  TypeCase(Type.Name("?"), Type.Name("Left"))
```

Comment:
This should be a wildcard still.

Reply:
not really. `case ?` is not a wildcard, at least not in my testing. unless the `?` type is defined earlier, the compiler complains. could you check, please?

Reply:
Actually, you are right, I didn't realize it.